docker-compose.prod.yml

```diff
@@ -44,14 +44,17 @@ services:
          gunicorn --bind 0.0.0.0:8080 "hubgrep:create_app()"
   
   sphinx:
-    image: macbre/sphinxsearch:3.3.1
+    image: macbre/sphinxsearch:3.4.1
     env_file: .env
     ports:
     - "127.0.0.1:9312:9312" # bind to local interface only!
     - "127.0.0.1:9306:9306"
     volumes:
     - ./sphinx_data:/var/lib/sphinx/data  # directory where sphinx will store index data
     - ./sphinx.conf:/opt/sphinx/conf/sphinx.conf  # SphinxSE configuration file
+    - ./sphinx_logs:/var/log
+    cap_add:
+      - IPC_LOCK
```

Comment:
linking the logs out, ipc_lock is needed for mlock (usually docker doesnt allow it)